open_dread_rando/files/schema.json

```diff
@@ -36,20 +36,10 @@
                     "resources": {
                         "type": "array",
                         "items": {
-                            "type": "object",
-                            "properties": {
-                                "item_id": {
-                                    "$ref": "#/$defs/item_id"
-                                },
-                                "quantity": {
-                                    "type": "number"
-                                }
-                            },
-                            "required": [
-                                "item_id",
-                                "quantity"
-                            ],
-                            "additionalProperties": false
+                            "type": "array",
```

Comment:
From reading the code, this should have a `"minItems": 1`.

And reading a bit more, there's a nested array here that I even missed. Would be great to have a description/minItems on both so it's a bit more obvious!